packages/web/components/complex/paginated-table.tsx

```diff
@@ -86,6 +86,7 @@ export const PaginatedTable = ({
               legacyBehavior
             >
               <a
+                target="_blank"
```

Comment:
this will make all pools open in a new tab, but only cosmwasm pools should open in a new tab since they link to celetone